tests/test_handlers.py

```diff
@@ -899,3 +932,32 @@ def test_lennard_jones_handler():
     # if a parameter is > 99 then its adjoint should be zero (converse isn't necessarily true since)
     mask = np.argwhere(params > 90)
     assert np.all(adjoints[mask] == 0.0)
+
+
+def test_symmetry_classes():
+    """Assert get_symmetry_classes returns arrays of expected length num_atoms"""
+    mols = fetch_freesolv()[:10]
+    for mol in mols:
+        sym_classes = get_symmetry_classes(mol)
+        assert len(sym_classes) == mol.GetNumAtoms()
+
+
+def test_spurious_param_idxs():
+    """Assert that get_spurious_param_idxs
+    * returns arrays with length sometimes > 0
+    * only ever returns indices of symmetric bond types"""
+    mols = fetch_freesolv()[50:55]  # expect three of these to trigger
```

Comment:
Specific idx into the results of fetch here seems kind of fragile, should these be retrieved by mol name or split off into a file specific for this test?

Reply:
Good call! Retrieved by name in https://github.com/proteneer/timemachine/pull/737/commits/0ab67964a592d8df1cec6eaf3be994b6d98f2b73